doc/source/installation/yum_repo.rst

```diff
@@ -18,6 +18,36 @@ Supported Releases:
 
 The *CentOS* repositories should work well with *Red Hat Enterprise Linux* too, provided that :program:`yum` is installed on the server.
 
+.. important::
+
+   *CentOS* 6 offers an outdated version of the ``curl`` library required by
+    the keyring Vault plugin of |Percona Server|. The version of the ``curl`` library in *CentOS* 6, which
+    depends on the ``nss`` library, is known to create memory corruption
+    issues. This bug is `registered in Red Hat Bugzilla
+    <https://bugzilla.redhat.com/show_bug.cgi?id=1057388>`_. Its current status
+    is `CLOSED WONTFIX`.
+
+    If you intend to use the keyring Vault plugin of |Percona Server|
+    make sure that you use the latest version of the ``curl`` library.
+    We recommend that you `build it from source
+    <https://curl.haxx.se/docs/install.html>`_ configuring with ssl:
+
+    .. code-block:: bash
+
+       $ ./configuration --with-ssl --prefix=<INSTALATION DIRECTORY>
```

Comment:
Also there was this (optionally --without-nss as nss was causing issues). Could be also "as NSS library was causing issues).

Reply:
Ready for review. The previous paragram explains that nss was causing issues. since this example is quite close to that text --without-nss should be self explanatory.